src/baseball/main.py

```
@@ -1,9 +1,83 @@
+            return 0
+
+def restart_baseball(): # 게임 종료 후 '1' 입력시 다시 시작, '2' 입력시 완전히 종료하는 기능 함수
+    print("게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.")
+    try:
+        user_key = int(input())
+        if user_key in [1, 2]:
+            return user_key
+        else:    # 잘못된 값을 입력할 경우 'ValueError'를 발생
+            raise ValueError("1 또는 2만 입력 가능합니다")
+    except ValueError:
+        raise ValueError("올바른 숫자를 입력해주세요 (1 또는 2)")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)